src/context/network.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { createContext, useContext, FunctionComponent } from "react";
+import { useNetInfo } from "@react-native-community/netinfo";
+
+interface NetworkContext {
+  isConnected: boolean;
```

Comment:
Wondering if isOnline will be a better name

Reply:
I'm following the naming here: https://github.com/react-native-community/react-native-netinfo/blob/master/README.md#netinfostate